packages/api/es/indexer.js

```diff
@@ -20,8 +20,60 @@ const { Search } = require('./search');
 const Rule = require('../models/rules');
 const uniqBy = require('lodash.uniqby');
 
-async function indexLog(payloads, index = 'cumulus', type = 'logs') {
-  const esClient = await Search.es();
+/**
+ * Returns the collectionId used in elasticsearch
+ * which is a combination of collection name and version
+ *
+ * @param {string} name - collection name
+ * @param {string} version - collection version
+ * @returns {string} collectionId
+ */
+function constructCollectionId(name, version) {
+  return `${name}___${version}`;
+}
+
+/**
+ * Returns the name and version of a collection based on
+ * the collectionId used in elasticsearch indexing
+ *
+ * @param {string} collectionId - collectionId used in elasticsearch index
+ * @returns {Object} name and version as object
+ */
+function deconstructCollectionId(collectionId) {
```

Comment:
Not worth changing, but this would be a cool place for array destructuring.

http://airbnb.io/javascript/#destructuring--array

```
const [name, version] = collectionId.split('___');
return { name, version };
```

Reply:
💯